pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -42,15 +42,15 @@
 from pyanaconda.i18n import _, P_, N_, C_
 from pyanaconda.bootloader import BootLoaderError
 
-from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX, CLEARPART_TYPE_NONE, AUTOPART_TYPE_LVM
+from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX, CLEARPART_TYPE_NONE
 from pykickstart.errors import KickstartValueError
 
 from collections import OrderedDict
 
 import logging
 log = logging.getLogger("anaconda")
 
-__all__ = ["StorageSpoke", "AutoPartSpoke"]
+__all__ = ["StorageSpoke"]
```

Comment:
In tui, it does not make sense to enable automatic import of indirect spokes, because we always create them and use them only in other spokes. With automatic import, they will be found by a hub, created, initialized and forgotten.

Also, I've added a new parameter to the `AutoPartSpoke.__init__` method and the initialization in a hub does not work anymore.